api/envoy/config/listener/v3/listener.proto

```diff
@@ -119,15 +119,14 @@ message Listener {
   // Configuration for filter chains discovery.
   // [#not-implemented-hide:]
   message FcdsConfig {
-    // Optional name to present to the filter chain discovery service. This may be an arbitrary name with arbitrary
-    // length. If a name is not provided, the listener's name is used. Refer to :ref:`filter_chains <envoy_v3_api_field_config.listener.v3.Listener.name>`.
-    // for details on how listener name is determined if unspecified. In addition, this may be a xdstp:// URL.
-    string name = 1;
+    // xdstp:// resource locator for the filter chain collection.
```

Comment:
Please add more notes here. Specifically what will be the constraints on the resource locator.

Note to other reviewers: this is probably not a breaking change as FCDS wasn't implemented anywhere.